File: Kotlin/프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.kt
class Solution {
    fun solution(s: String): String {

        val arr = s.split(" ")

        val sb = StringBuilder()

        for (s1 in arr) {
            for (c in s1.indices) {

                var char = s1[c]

                if (c % 2 == 0) {
                    sb.append(char.uppercaseChar())
                    continue
                }
                sb.append(char.lowercaseChar())
            }
            sb.append(" ")
        }

        return sb.replace(sb.length - 1, sb.length, "").toString()
    }
}

File: C#/프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.cs
using System.Text;

public class Solution {
    public string solution(string s) {
        string answer = "";
        
        string[] spl = s.Split(' ');
        
        for(int j = 0; j < s.Length; j++)
        {
            string s1 = spl[j];
            
            for(int i = 0; i < s1.Length; i++)
            {
                if (i % 2 == 0) 
                {
                    if (97 <= s1[i] && s1[i] <= 122)
                    {
                        answer += ((char)(s1[i]-32)).ToString();
                    }
                    else
                    {
                        answer += s1[i];
                    }
                }
                else
                {
                    if (97 <= s1[i] && s1[i] <= 122)
                    {
                        answer += s1[i];
                    }
                    else
                    {
                        answer += ((char)(s1[i] + 32)).ToString();
                    }
                }
            }
            
            if (j == spl.Length - 1)
                break;
            
            answer += " ";
        }
        
        return answer;
    
    }
}